### ISSUE TITLE
Electron crashes when we enable proxy and devTools

### ISSUE THREAD

**Issue by berdof** [segmentio/nightmare#578]
I start nightmare with the following snippet.
```
 Nightmare({
    switches: {
      'proxy-server': 'socks5://0.0.0.0:9150',
      'ignore-certificate-errors': true
    },
    paths: {
      userData: './cookies/' + port
    },
    openDevTools: true,
    show: true
  });
```
But unfortunately it crashes when calling `nightmare.goto('http://google.com')` because of      `openDevTools: true`. When I comment this line everything's working perfect.

**Comment by dwelle** [comment]
confirming it crashes on `openDevTools` (running no proxy server), v `2.3.2`.

**Comment by rosshinkley** [comment]
This is a problem with Electron: electron/electron#5180

Related: #563, #572, and an attempt at fixing in #576.

**Comment by rosshinkley** [comment]
Should be fixed with #588.  Closing.
